src/examples/java/io/nats/examples/jetstream/NatsKeyValueFull.java

```diff
@@ -0,0 +1,209 @@
+// Copyright 2020 The NATS Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package io.nats.examples.jetstream;
+
+import io.nats.client.*;
+import io.nats.client.api.BucketConfiguration;
+import io.nats.client.api.BucketInfo;
+import io.nats.client.api.KvEntry;
+import io.nats.client.api.StorageType;
+import io.nats.examples.ExampleArgs;
+import io.nats.examples.ExampleUtils;
+
+/**
+ * This example will demonstrate Key Value store.
```

Comment:
I kind of want to design the example so all clients do the exact same example with exact same cli.

Then we can make an automated integration test where CI can test that CLI / Java / Go / .Net / JS all can interoperate by calling the various examples at different tmes - like make bucket with java, write value, verify CLI gets the expected value out etc

Just a thought no changes now.